index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+
+    <header class="header flex-row">
+        <div class="menu-button">
+            <a class="logo">JEPM</a>
+        </div>
+        <nav class="menu-mobile">
+            <button type="button" class="menu-button"><img src="./img/header-menu.png" alt="menu-image"></button>
+        </nav>
+    </header>
+    <section class="headline main">
+        <h1 class="presentation"> I'm Jorge E. Prada M.<br>
+            Glad to see you!<br>
+        </h1>
+        <p class="color-text">I'm a software developer! I can help you build a product, feature or website Look
+            through some of my work and
+            experience! If you like what you see and have a project you need coded,
+            don't hesitate to contact me.</p>
+        <div class="text-connect">
```

Comment:
As previously mentioned, I suggest using flex to align this content to the center so that the design looks like the figma design. Consider aligning the icons to the right, using flexbox to make the space at the top and bottom match the design of figma.

Your design

<img width="215" alt="ui" src="https://user-images.githubusercontent.com/84930355/235001384-005c02ad-8937-45e0-a404-15a364354df1.PNG">


Figma 

<img width="176" alt="ui" src="https://user-images.githubusercontent.com/84930355/235001948-5045f433-8fad-4247-b89e-25b482c8a3c3.PNG">

Reply:
The recommended modifications were made.
![image](https://user-images.githubusercontent.com/10118922/235040808-e51a1cf2-e526-49c6-bc8a-3b9f4e3a7a44.png)